linz/2014/script-hsi.pro

```diff
@@ -1,3 +1,98 @@
+;
+; Using Säm's RHESSI image.
+;
+
+restore,'sam/rhsi_xrt_last_foxsi_target.dat'
+loadct2,3
+;popen
+!p.multi=[0,2,1]
+this_fov=3
+this_center=[20,-70]
+this_center=[-65,90]
+this_level=[50,70,90]
+this_xrt=gxmap23
+this_shift=[23,24]
+this_xrt.xc=this_xrt.xc+this_shift(0)
+this_xrt.yc=this_xrt.yc+this_shift(1)
+
+plot_map,this_xrt,fov=this_fov,center=this_center,bot=9,dmax=-1700,dmin=-2500
+plot_map,h_arx,/over,/per,levels=this_level,color=2,thick=4
+xyouts,-147,19,'RHESSI 4-8 keV',charsize=1.4,color=2
+xyouts,-147,7,'19:18:50-19:19:14UT',charsize=1.4,color=2
+
+this_xrt2=gxmap24
+this_xrt2.xc=this_xrt2.xc+this_shift(0)
+this_xrt2.yc=this_xrt2.yc+this_shift(1)
+plot_map,this_xrt2,fov=this_fov,center=this_center,bot=9,dmax=-1700,dmin=-3200
+plot_map,h_arx,/over,/per,levels=this_level,color=2,thick=4
+xyouts,-147,19,'RHESSI 4-8 keV',charsize=1.4,color=2
+xyouts,-147,7,'19:18:50-19:19:14UT',charsize=1.4,color=2
+;pclose
+
+;
+; end Säm's script.
+;
+
+; Adding on the FOXSI image too.
+
+trange=[t5_start, t5_end]
+cen = cen5
+en = [4.,8.]
+m = foxsi_image_map( data_lvl2_d6, cen, erange=en, trange=trange, /XYCORR, thr_n=4., smooth=2 )
+m.id = 'FOXSI Det6 4-8 keV'
+
+loadct, 5
+plot_map, m, cen=this_center, fov=3
+
+sub_fox = make_submap( m, cen=this_center, fov=3)
+sub_hsi = make_submap( h_arx, cen=this_center, fov=3)
+print, map_centroid(sub_fox, thr=1.)
+print, map_centroid(sub_hsi, thr=0.0)
+IDL> print, map_centroid(sub_fox, thr=1.)
```

Comment:
In the LINZ directory, you'll find most of the scripts I use for analysis, for reference, so that others can see what I'm doing.  I don't guarantee any of this code, especially since some of it is quite old.  The most useful code in each script is likely to be at the bottom (i.e. newest).  For software documentation and examples of how to use routines, see the HELP directory, not this one.